lib/sidekiq/extensions/manager.rb

```diff
@@ -1,12 +1,11 @@
 class Sidekiq::Manager
   module InitLimitFetch
-    def initialize(options={})
+    def initialize(capsule_or_options)
       if Sidekiq::LimitFetch.post_7?
-        options.config[:fetch] = Sidekiq::LimitFetch
+        capsule_or_options.config[:fetch_class] = Sidekiq::LimitFetch
```

Comment:
If I'm not wrong, starting from Sidekiq v7, the option to define a custom "fetcher" is `fetch_class`, cf https://github.com/mperham/sidekiq/blob/4d3f90ba56dd9f9faa35a18bbdfb1b5cb726ee5c/lib/sidekiq/capsule.rb#L37